### ISSUE TITLE
Jinux randomly crashes

### ISSUE THREAD

**Issue by Atomicbeast101** [Atomicbeast101/Jinux#20]
Crash log: https://i.gyazo.com/c5af6e039aaa3646266edc9b3765f6f2.png

**Comment by AlexHoorn** [comment]
@Atomicbeast101 Please provide the config file alongside the version you're running.

**Comment by Atomicbeast101** [comment]
@AlexHoorn Version 3.0. Config.ini file:
```Ini
[Jinux]
channel = -REMOVED-
character = -
client_id = -REMOVED-
playing = -help | Jinux v3.0
token = -REMOVED-
logging = On
auto_welcome = On
auto_welcome_channel = -REMOVED-

[Twitch]
enabled = True
interval = 300
users = username,username
channel = -REMOVED-

[Functions]
eightball = On
channelinfo = On
choose = On
chucknorris = On
cleverbot = On
coinflip = On
currency = On
dice = On
dictionary = On
poll = On
random_cat = On
random_gif = On
purge = Off
reddit = On
rock_paper_scissors = On
serverinfo = On
temperature = On
timezone = On
translate = On
update = On
xkcd = On
youtube = On
```

**Comment by Atomicbeast101** [comment]
Update on error: https://i.gyazo.com/921833fbf08d3460dc02c0daa726ed02.png

**Comment by Atomicbeast101** [comment]
Issue resolved in v4.0 release.
